inference-engine/thirdparty/movidius/XLink/shared/src/XLinkDispatcher.c

```diff
@@ -24,6 +24,7 @@
 # include "win_semaphore.h"
 #else
 # include <pthread.h>
+# include <unistd.h>
```

Comment:
Shouldn't we put this under ifdef __APPLE__, or it's required for Linux as well?

Reply:
It won't hurt Linux:)